src/fixed/layout.rs

```diff
@@ -14,26 +14,27 @@ pub(crate) enum LayoutModifiers {
     AltGr,
 }
 
-pub(crate) struct LayoutParser {
-    layout: HashMap<String, String>,
+/** This was renamed to `Layout` from `LayoutParser`*/
```

Comment:
How about describing its functions?
```rust
/// Layout Parser
/// 
/// Parses fixed method layout file(JSON formatted) so the character for a specific key can be inquired.
```
Improvements are welcome!

Reply:
The idea of renaming `LayoutParser` to `Layout` came to me because, it felt the primary function of the type is to provide the mapping between the user inputs and output characters. As the "parsing" from JSON to HashMap is now done only on initialization, it is not currently a primary function of the type IMHO. Does this sound right?

I'll fix the comment format, though. But how should we describe this type? What should be in the comment?

Reply:
> As the "parsing" from JSON to HashMap is now done only on initialization, it is not currently a primary function of the type IMHO. Does this sound right?

Yeah, though parsing is now done only on initialization, it's not bad to mention it in the docs from my perspective as we are providing a JSON value and it is producing a map to look up characters for keys. But you can leave it out if sounds redundant to you. 

> But how should we describe this type? What should be in the comment?

Umm, I think just mentioning its functionalities would suffice, as you said in your reply.